Find a functionally equivalent code:android.widget.DatePicker.setCalendarViewShown:COMMENT
Method Modifier: public      
Comment:/**
 * Sets whether the {@link CalendarView} is shown.
 *
 * @param shown True if the calendar view is to be shown.
 */

@@ -1,3 +1,3 @@
 {
-    mCalendarView.setVisibility(shown ? VISIBLE : GONE);
+    mDelegate.setCalendarViewShown(shown);
 }

